ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/DistributingExecutor.java

```diff
@@ -48,28 +42,31 @@
  * {@code distributedCmdResponseTimeout}.
  */
 public class DistributingExecutor {
```

Comment:
Don't feel too strongly about this but we could have DistributingExecutor implement the StatementExecutor interface again by keeping the properties variable (it's just unused) as a parameter in the `execute()` method.